packages/libs/react-ui/src/components/Table/Th.tsx

```diff
@@ -6,12 +6,13 @@ import classNames from 'classnames';
 import type { FC } from 'react';
 import React from 'react';
 
-export interface IThProps extends Pick<Sprinkles, 'width'> {
+export interface IThProps
```

Comment:
Out of curiosity; why don't we (also) extend the props of the (React's) `th` element? (and then spreading those props on the element)

Reply:
Hmm not sure. Maybe it just hasn't needed so it wasn't implemented, but I do think it would be valuable. I've added it to one of the tickets we are refining today